### ISSUE TITLE
Can't see Icons

### ISSUE THREAD

**Issue by ZetoxZ** [shoestare/EavesDropClassic#1]
I was unable to see any icons next to the Text for spells/buffs/debuffs etc. Is this function going to be enabled? Thank you :)

**Comment by ryantkerr** [comment]
@ZetoxZ yes it will be fixed. I wasn't able to get to it during the stress test unfortunately, but it should be fixed shortly after launch.

**Comment by ZetoxZ** [comment]
Awesome!!!! thanks so much! 

Also one more thing: If it's not too much to ask--
Could you add an option to allow (Shift + Left click) to immediately be sent to chat instead of being pasted to the chat window? (Currently it was needing to be sent by pressing Enter). Many times it's extremely useful just to hit Shift + Left Click a few times instantly to send to others in chat my combat log in chronological order. 

That would be wonderful! Appreciate all your work here

**Comment by ZetoxZ** [comment]
Any update?

**Comment by 62574353** [comment]
Would love to see this get fixed. EavesDrop is great, but the icons are so essential!

**Comment by johanruda** [comment]
> Would love to see this get fixed. EavesDrop is great, but the icons are so essential!

Agreed! I hope they'll be back soon! :)

**Comment by Hashmire** [comment]
+1 would really love to see icons, was very excited when I saw this got ported

**Comment by lvangh** [comment]
Was just going to post a topic about this - yeah in 1.12 you could see the icons. 

![image](https://user-images.githubusercontent.com/19690096/68817145-9d480e80-0635-11ea-9341-d7e82bf449f6.png)

**Comment by ghost** [comment]
As pointed out, here is the fix for missing icons:
https://github.com/ryantkerr/EavesDrop-Classic/pull/2/commits/6d6d8c176ff5e0a1501d192d690c8947a6c9b919
